site/content/en/docs/tasks/run/multikueue/deployment.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Run Deployment in Multi-Cluster"
+linkTitle: "Deployment"
+weight: 2
+date: 2025-02-17
+description: >
+  Run a MultiKueue scheduled Deployment.
+---
+
+## Before you begin
+
+1. Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+2. For the ease of setup and use we recommend using at least Kueue v0.10.2.
```

Comment:
Pod MK adapter will be in v0.10.2 which is being released in March I believe?

Reply:
I think you mean 0.11.0?

I don't think there are plans to cherry-pick this in a release yet.

Reply:
Yes, 0.11.0 will be the first to include the adapter. We don't typically backport features, unless there are good reasons, but in this case I think we can wait.